tests/integration/test_get_ontology_type.py

```diff
@@ -46,6 +47,34 @@ async def test_get_ontology_type(client: Any, fs: Any) -> None:
     assert text == expected
 
 
+@pytest.mark.integration
+async def test_get_ontology_type_that_does_not_exist(client: Any, fs: Any) -> None:
+    """Should return status 200 OK and html document."""
+    expected = dedent(
+        """
+        <!doctype html>
+        <html lang="en">
+            <title>Not found</title>
+            <body>
+                <p>The page you are looking for does not exist.</p>
+            </body>
+        </html>
+    """
+    ).strip()
+
+    ontology_type = "does_not_exist"
+
+    headers = {hdrs.ACCEPT: "text/html"}
+    response = await client.get(f"/{ontology_type}", headers=headers)
+
+    assert response.status == 404
+    assert "text/html" in response.headers[hdrs.CONTENT_TYPE]
+    assert "en" in response.headers[hdrs.CONTENT_LANGUAGE]
```

Comment:
Er `response.headers[hdrs.CONTENT_TYPE]` en list eller string?

Om det er en liste forventer vi ett element, og exact match:
```suggestion
    assert ["text/html"] == response.headers[hdrs.CONTENT_TYPE]
    assert ["en"] == response.headers[hdrs.CONTENT_LANGUAGE]
```

om det er én value per header forventer vi exact match:
```suggestion
    assert "text/html" == response.headers[hdrs.CONTENT_TYPE]
    assert "en" == response.headers[hdrs.CONTENT_LANGUAGE]
```

Reply:
Fiksa i 2dc0527f4da4b4f95d511b5e0ad63375d7953618